### PR TITLE
Add log ticks to log binned barchart

### PR DESCRIPTION
**Pull request by sharph**
Going to see what Jeremy thinks of this when he gets in.

logticks

### CODE REVIEW

**Comment by sharph** [comment]
logticks2

**Comment by CloudNiner** [comment]
Code LGTM

**Comment by sharph** [comment]
@CloudNiner This could probably use another look after what I added.

@jerheff 
logticks3

File: app/scripts/charting/barchart-directive.js [hunk 1]
```diff
@@ -130,9 +132,14 @@
         }
 
         module.link = function ($scope, element, attrs) {
+            // http://bl.ocks.org/mbostock/6738109
+            var superscript = '⁰¹²³⁴⁵⁶⁷⁸⁹',
```
Comment (CloudNiner) [inline app/scripts/charting/barchart-directive.js]: One variable per line.

File: app/scripts/charting/barchart-directive.js [hunk 2]
```diff
@@ -130,9 +132,14 @@
...
+                formatPower = function(d) { return (d + '').split('').map(function(c) { return superscript[c]; }).join(''); };
```
Comment (CloudNiner) [inline app/scripts/charting/barchart-directive.js]: Format this better so the line doesn't run on.

Comment (CloudNiner) [inline app/scripts/charting/barchart-directive.js]: Clever way to get any power though from an integer, though.

Comment (sharph) [inline app/scripts/charting/barchart-directive.js]: Yeah, credit has to go to http://bl.ocks.org/mbostock/6738109 for that.

**Comment by CloudNiner** [comment]
+1 after formatting cleanup

**Pull request merged**